### ISSUE TITLE
Dukpy leaks memory

### ISSUE THREAD

**Issue by maximinus** [amol-/dukpy#33]
Tested with Python 3.5.3 from master branch

The issue can be seen with:

```
import dukpy
import psutil

for i in range(100):
    dukpy.evaljs("var o = {'value': 5}; o['value'] += 3; o")
print(psutil.virtual_memory())
```

Increase the value 100 to see larger leaks.

**Comment by carsonyl** [comment]
`for i in range(100): dukpy.JSInterpreter()` is enough to demonstrate a leak.

**Comment by amol-** [comment]
Released 0.2.2 with the fix for this.
